pandas/core/series.py

```diff
@@ -2820,7 +2825,7 @@ def _try_kind_sort(arr):
                 return arr.argsort(kind="quicksort")
 
         arr = self._values
-        sortedIdx = np.empty(len(self), dtype=np.int32)
+        sorted_index = np.empty(len(self), dtype=np.int32)
```

Comment:
Orthogonal to this but I wonder why this is specified as `np.int32`; might be a bug for input with more than 2 ** 32 - 1 rows

Reply:
numpy uses int32 for argsort so best can do (to be honest 2B is enough)